modules/microshift-data-cleaning-certs.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-cleanup-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-data-cleaning-certs_{context}"]
+= Cleaning the certificates data
+
+Run the following command to cleanup {microshift-short} certificates and to reset {microshift-short} custom certificates so they recreate when the {microshift-short} service restarts. This command performs the following cleanup actions:
```

Comment:
This sentence is not correct. Please, do not change the sense of what's written in the doc.